### ISSUE TITLE
Add `rejects` to `uvu/asset`

### ISSUE THREAD

**Issue by ai** [lukeed/uvu#155]
What do you think about adding very popular matcher to `uvu/asset`? Like `throws`, but for `async` function.

```js
rejects(() => asyncFn(), 'Some error')
```

Or we can add `async` support to `throws` when `cb` returns `Promise`.

```js
throws(async () => asyncFn(), 'Some error')
```

**Comment by lukeed** [comment]
Duplicate of #131 and implemented in #132 🎉
It's going to be part of the next minor release :)

**Comment by Heartnett** [comment]
> Duplicate of #131 and implemented in #132 🎉 It's going to be part of the next minor release :)

Waiting for this feature to drop...
